libcudacxx/docs/ptx/instructions/elect.sync.md

```diff
@@ -0,0 +1,20 @@
+# elect.sync
+
+- PTX ISA: [`elect.sync`](https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#parallel-synchronization-and-communication-instructions-elect-sync)
+
+*Implementation note:* Since C++ does not support returning multiple values, the
+variant of the instruction that returns both a predicate and an updated
+membermask is not supported.
```

Comment:
I believe we should just define a `struct elect_syncc_return` and use that

If we can make sure that it supports structured bindings, even better

Reply:
There are a couple of other reasons I didn't expose it:
1. Nobody seems to use the member mask (e.g.: [CUTLASS throws it away](https://github.com/NVIDIA/cutlass/blob/ffa34e70756b0bc744e1dfcc115b5a991a68f132/include/cute/arch/cluster_sm90.hpp#L179), and cooperative_groups::invoke_one does the same).
2. I can't think of a usecase either.
3. Codegen would become terribly complicated.

Reply:
Fine by me